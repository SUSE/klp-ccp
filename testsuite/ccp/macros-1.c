#define A

void pu_f(void)
{
	A
}
