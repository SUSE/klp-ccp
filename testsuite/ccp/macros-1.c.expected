/* klp-ccp: from testsuite/ccp/macros-1.c */
#define A

void klpp_pu_f(void)
{
	A
}
